pkgs/by-name/po/powersploit/package.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "PowerSploit";
+  version = "v3.0.0";
```

No prefix for our version.

```suggestion
  version = "3.0.0";
```